eap6-support/hawkular-javaagent-wildfly-feature-pack-eap6/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml

```diff
@@ -750,6 +750,11 @@ resource-type-set-dmr:
     - name: Base Directory
       attribute: base-dir
       path:      /core-service=server-environment
+    operation-dmr:
+    - name: JDR
+      internal-name: generate-jdr-report
+      modifies:      true
```

Comment:
I think you are right, it should be a read-only function.
I removed that property from Domain Wildflies and also for standalone ones.